### ISSUE TITLE
make it available on NPM

### ISSUE THREAD

**Issue by infacq** [ImperdibleSoft/material-css#4]

**Comment by lordfido** [comment]
Done!

npm install materialcss

**Comment by infacq** [comment]
thanks

**Comment by bitifet** [comment]
That's great, thanks!

But this only puts files on node-modules/materialcss/material-css directory which is not usually available at client-side.

Of course, I can link it from public directory (having static an router mounted over there) but it is a bit tricky anyway.

I looked at module's index.js and saw that it does nothing (except exporting empty object).

That arises to me a cool (but probably not as "generic" as you would like to adopt) idea consisting of providing express static router serving those files.

This doesn't break anything but requires to add a dependency over the express-static module which I'm not sure if you would like to rely on. So I opted to build my own independent module requiring both (materialcss and express-static) as dependencies.

If it could be useful to anyone, it's available at [Github](https://github.com/bitifet/ematerial) and [npm](https://www.npmjs.com/package/ematerial) (npm install ematerial).

...but, on the other hand, if you like it, feel free to merge into your npm module and notify me to remove (or replace with a backlink) my module from npm and github.
